dependencies/analyses/nm-symbols-analysis.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+
+if test "$#" -ne 1; then
+	echo "invalid arguments: usage: $0 path/to/binary"
+	exit 1
+fi
+
+DATA=$(go tool nm -size "$1" \
+	| awk 'NF==4 {printf "%s\t%s\t%s\n", $2, $3, $4}' \
+	| grep -v -P "\t_\t" \
+	| grep -P "\tt\t" \
+	| awk ' {printf "%s\t\t%s\n", $1, $3} ' \
+	)
```

Comment:
Might this be rewritten more simply?
```
go tool ... | awk 'NF==4 && $3=="t" { printf "%s\t\t%s\n", $2, $4}'
```
(and again, I think it might read more easily without the temporary variable)

Reply:
Much nicer, thanks!